thorlcr/thorutil/thmem.cpp

```diff
@@ -214,6 +227,7 @@ class CSpillableStreamBase : public CSimpleInterface, implements roxiemem::IBuff
     	assertex(inRows.isFlushed());
```

Comment:
it's a tab, there were a couple of other lines with tab's in the file too, have corrected.